queries/markdown/highlights.scm

```diff
@@ -34,6 +34,29 @@
   (info_string (language) @conceal
   (#set! conceal "")))
 
+;; Conceal blockquote markers
+((block_quote_marker) @punctuation.special
+  (#offset! @punctuation.special 0 0 0 -1)
+  (#set! conceal "▐"))
+((block_continuation) @punctuation.special
+  (#eq? @punctuation.special ">")
+  (#set! conceal "▐"))
+((block_continuation) @punctuation.special
+  (#eq? @punctuation.special "> ")
+  (#offset! @punctuation.special 0 0 0 -1)
+  (#set! conceal "▐"))
+((block_continuation) @punctuation.special
+  ; for indented code blocks
+  (#eq? @punctuation.special ">     ")
+  (#offset! @punctuation.special 0 0 0 -5)
+  (#set! conceal "▐"))
+
+;; Conceal bullet points
+([(list_marker_minus) (list_marker_plus) (list_marker_star)]
+  @punctuation.special
+  (#offset! @punctuation.special 0 0 0 -1)
+  (#set! conceal "•"))
```

Comment:
This will lead to some confusion, as there's a reason after all to have `-`, `+` and `*` list markers. Forcing GFM-style markers here is not favorable imo

Reply:
I see. Which node(s) do you think this query should apply to?

Reply:
imo, There's an argument for any of the 3 to be applied by this query, just not all 3.

Reply:
My vote would be `-` (or `-` and `*`), I guess we'll see what others say